storage/aptosdb/src/transaction_store/mod.rs

```diff
@@ -92,7 +72,7 @@ impl TransactionStore {
     ) -> Result<AccountTransactionVersionIter> {
         let mut iter = self
             .ledger_db
-            .transaction_db()
```

Comment:
Since we are doing this refactoring, would it simply the code more if we completely remove the transaction_store and move this to either TransactionDB or LedgerStore ?

Reply:
The functions in the TransactionStore after this PR are considered deprecated, and will be removed all together when sharding is enabled.

Reply:
this function is used in aptos_db_reader. you want to stop providing this iterator or implementing the same function in a different way?

Reply:
we won't have the transaction by account index anymore, the functionality will go to indexer side.